src/Common/src/Interop/Windows/kernel32/Interop.FormatMessage_SafeLibraryHandle.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        public const int FORMAT_MESSAGE_IGNORE_INSERTS = 0x00000200;
+        public const int FORMAT_MESSAGE_FROM_HMODULE = 0x00000800;
+        public const int FORMAT_MESSAGE_FROM_SYSTEM = 0x00001000;
+        public const int FORMAT_MESSAGE_ARGUMENT_ARRAY = 0x00002000;
+        public const int ERROR_INSUFFICIENT_BUFFER = 0x7A;
+
+        [DllImport(Libraries.Kernel32, CharSet = CharSet.Auto, SetLastError = true, BestFitMapping = true)]
+        public static unsafe extern int FormatMessage(
+            int dwFlags,
```

Comment:
Is there a reason the existing FormatMessage function cannot be used? Also is there a reason you didn't put this in that file we generally put the overloads in the same interop file.

Reply:
The format message for this class requires a SafeLibraryHandle parameter inplace of integer_ptr 
The new formatmessage required two more reference dependencies which then will need to be included in all the projects using the  earlier formatmessage.cs (which are a quite bunch of them)
So Earlier I put the overloaded function in the same file but then Dan suggested to make a new file.